kratos/mpi/python/add_mpi_utilities_to_python.cpp

```diff
@@ -76,6 +76,7 @@ void AddMPIUtilitiesToPython(pybind11::module& m)
     ;
 
     py::class_<GatherModelPartUtility>(m, "GatherModelPartUtility")
+        .def(py::init<int, ModelPart&, int, ModelPart&>())
```

Comment:
?

Reply:
The same constructor, sorry, I will correct